packages/frontend/core/src/modules/collection-rules/impls/filters/tags.ts

```
@@ -15,23 +16,66 @@
-      const tagIds = params.value?.split(',') ?? [];
-
-      const tags = tagIds.map(id => this.tagService.tagList.tagByTagId$(id));
+    const method = params.method as WorkspacePropertyFilter<'tags'>;
+    const tagIds = params.value?.split(',') ?? [];
+    const tags = tagIds.map(id => this.tagService.tagList.tagByTagId$(id));
+    if (method === 'include-all' || method === 'not-include-all') {
+      if (tags.length === 0) {
+        return of(new Set<string>());
+      }
 
+      const includeDocIds$ = combineLatest(tags).pipe(
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)